proto/types/dymint/state.proto

```diff
@@ -51,6 +51,7 @@ message State {
 
   int64 revision_start_height = 21;
 
+  uint32 prev_drs = 23;
```

Comment:
what is this for ?

Reply:
nothing, it was from previous test. removed